Dockerfile

```diff
@@ -18,11 +18,21 @@ COPY package*.json ./
 
 RUN npm install
 
+
+# 
+FROM base as dev
+
+COPY . .
+
+EXPOSE 8888
+
+#
+FROM base as prod
+
 # Bundle app source
 COPY . .
 
 # build
 RUN npm run build
 
 EXPOSE 8888
-CMD [ "npm", "run", "start:prod" ]

```

Comment:
What's starting the application if this is removed?

Reply:
I left the entrypoint call in compose, because in this case I left the environment thought of as a development environment